openapi.yaml

```diff
@@ -324,22 +324,27 @@ paths:
                     id: 6aca4yumen9yh
         description: 'If you''re sending in the first message of a conversation, you should send in a null conversationId and a null parentMessageId. This first call will generate a new conversation, and you will receive an id for the conversation in the response body.'
     get:
+      parameters:
```

Comment:
I think there are other places where `applicationId` should be moved to `parameters` as well, but didn't want to add too much noise to this PR.